computers/init.lua

```diff
@@ -93,6 +93,7 @@ end
 
 local MODPATH = minetest.get_modpath("computers")
 dofile(MODPATH.."/computers.lua")
-dofile(MODPATH.."/recipes.lua")
 dofile(MODPATH.."/gaming.lua")
 dofile(MODPATH.."/aliases.lua")
+
+if minetest.get_modpath("default") and minetest.get_modpath("basic_materials") then dofile(MODPATH.."/recipes.lua") end
```

Comment:
Should be on multiple lines.
```suggestion
if minetest.get_modpath("default") and minetest.get_modpath("basic_materials") then dofile(MODPATH.."/recipes.lua") end
```
```suggestion
if minetest.get_modpath("default") and minetest.get_modpath("basic_materials") then
	dofile(MODPATH.."/recipes.lua")
end

```

Reply:
this is a pointless loc increase for a simple thing under luachecks default line length limit

Reply:
Nothing to do with line length, all to do with readability.

And readability also rules over loc.